src/math/quat.js

```diff
@@ -372,15 +372,29 @@ class Quat {
     /**
      * Sets a quaternion from Euler angles specified in XYZ order.
      *
-     * @param {number} ex - Angle to rotate around X axis in degrees.
+     * @param {number|Vec3} ex - Angle to rotate around X axis in degrees. If ex is a Vec3,
+     * the Vec3 will be used to populate the first 3 components.
      * @param {number} ey - Angle to rotate around Y axis in degrees.
      * @param {number} ez - Angle to rotate around Z axis in degrees.

```

Comment:
You now need to mark `ey` and `ez` as optional.

Reply:
I noticed many optional parameters in functions such as https://github.com/playcanvas/engine/blob/main/src/resources/container.js#L158 are indicated by:
1. wrapped by square brackets in their name
2. having the word "optional" in the beginning of their comments

The functions in `graph-node.js` that I made changes to have indicator 1 but not indicator 2 for their optional parameters. Should I indicator 2 to ensure consistency? (in addition to marking `ey` and `ez`)

https://github.com/playcanvas/engine/blob/7e5ddc0e93cf3f054d0547a61b3d6af46c035eea/src/scene/graph-node.js#L906-L907

https://github.com/playcanvas/engine/blob/7e5ddc0e93cf3f054d0547a61b3d6af46c035eea/src/scene/graph-node.js#L1130-L1131

https://github.com/playcanvas/engine/blob/7e5ddc0e93cf3f054d0547a61b3d6af46c035eea/src/scene/graph-node.js#L1540-L1541

https://github.com/playcanvas/engine/blob/7e5ddc0e93cf3f054d0547a61b3d6af46c035eea/src/scene/graph-node.js#L1579-L1580